azure-mgmt-appservice/src/test/resources/session-records/canCRUDFunctionApp.json

```diff
@@ -189,7 +189,7 @@
     }
   }, {
     "Method" : "POST",
```

Comment:
just curious, what's the query param `expand=kerb` used for

Reply:
kerb likely stands for "Kerberos keys", to include Kerberos keys in the response (when the storage account in AD identity-based authentication).

Personally never configured that...

Reply:
kerb likely stands for "Kerberos keys", to include Kerberos keys in the response (when the storage account in AD identity-based authentication).

Personally never configured that...